synapsis/src/main/java/com/soen/synapsis/appuser/connection/ConnectionRepository.java

```diff
@@ -4,8 +4,12 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface ConnectionRepository extends JpaRepository<Connection, ConnectionKey> {
 
+    List<Connection> findAllByIdRequesterIDAndPendingIsFalse(Long requesterID);
```

Comment:
This name should probably less descriptive of the process and more descriptive of why we would use it. Something like findAcceptedConnectionsByRequesterID

Reply:
Done